tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -1341,3 +1341,41 @@ TEST(caffe2, sparseToDense) {
   // Graph has three inputs and one output.
   EXPECT_EQ(mod.getPlaceholders().size(), 4);
 }
+
+/// Test loading NCHW2NHWC op.
+TEST(caffe2, testNCHW2NHWC) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename(
+      "tests/models/caffe2Models/NCHW2NHWC_predict_net.pbtxt");
+  std::string NetWeightFilename(
+      "tests/models/caffe2Models/empty_init_net.pbtxt");
+
+  Placeholder *output;
+  Context ctx;
+
+  Tensor inputs(ElemKind::FloatTy, {1, 2, 3, 4});
+
+  // Destroy the loader after the graph is loaded since the following execution
+  // will not depend on anyting from the loader.
+  {
+    Caffe2ModelLoader caffe2LD(NetDescFilename, NetWeightFilename, {"inputs"},
+                               {&inputs.getType()}, *F);
+    output = caffe2LD.getSingleOutput();
+  }
+
+  // High level check on the content of the graph. We have 1 transpose and 1
```

Comment:
I believe checking the output shape in scope here for these tests, correct? If so can you add a check that the output shape is as expected (`{1,3,4,2}`)?

Reply:
Added the shape check.